packages/memory-storage/test/fs-fallback.test.ts

```diff
@@ -69,16 +69,20 @@ describe('fallback to fs for reading', () => {
         });
     });
 
-    test('attempting to read "other" key value store with no "__metadata__" present should read from fs, even if accessed without generating id first', async () => {
-        const otherStore = storage.keyValueStore('other');
-
-        const input = await otherStore.getRecord('INPUT');
-        expect(input).toStrictEqual<KeyValueStoreRecord>({
-            key: 'INPUT',
-            value: { foo: 'bar but from fs' },
-            contentType: 'application/json; charset=utf-8',
-        });
-    });
+    test(
+        'attempting to read "other" key value store with no "__metadata__" present should read from fs, ' +
+            'even if accessed without generating id first',
```

Comment:
oof. I am not a fan of this string split personally, even if the formatter does it... Is there no setting to disable that? 😢

Reply:
The formatter actually leaves strings be, however long they are - that split is mine. I did it to make `eslint` happy. We could of course disable the max line length rule...

Reply:
Drop the eslint rule completely. Also the operator should be on the next line in the beginning.